test/ERC721A.test.js

```diff
@@ -18,6 +18,11 @@ describe('ERC721A', function () {
       const supply = await this.erc721a.totalSupply();
       expect(supply).to.equal(0);
     });
+
+    it('has 0 totalMinted', async function () {
```

Comment:
Can you add totalMinted test cases for ERC721ABurnable?

Reply:
I add one here:  
https://github.com/chiru-labs/ERC721A/blob/f0a5bb42f85300e9857932c5ddb79164dd07b124/test/extensions/ERC721ABurnable.test.js#L56-L63  
Are there any other situations?

Reply:
This is hard. 

I'm thinking of simply adding a `directBurn` function to `ERC721AMock` if we want to test `_totalMinted` with `_burn` and `_startTokenId`. 

The route of having a `ERC721ABurnableStartOne`, or making `ERC721AMock` subclass `ERC721ABurnable` feels uncomfortable.

Reply:
Would be cool to test these features individually, however, these are quite connected. 
So I created a new mock combining both in one of my recent commits:
[`e16a79a` (#66)](https://github.com/chiru-labs/ERC721A/pull/66/commits/e16a79a5bc8382aef740cc843e3c4d04a3ff03d9)